tests/codex/blockexchange/engine/testblockexc.nim

```diff
@@ -233,9 +205,9 @@ asyncchecksuite "NetworkStore - multiple nodes":
       )
 
     for i in 0..15:
-      (await networkStore[i div 4].engine.localStore.putBlock(blocks[i])).tryGet()
+      (await nodes[i div 4].networkStore.engine.localStore.putBlock(blocks[i])).tryGet()
```

Comment:
I'm lost on this bit, why the switch from engine to networkStore here?

Reply:
I'm creating nodes instead of individual parts now, the change is behavior-preserving.

Reply:
We're accessing engine in both versions, the difference is that before we had a reference to an array of networkStore, now we have a reference to an array of nodes (hence need to do nodes[blabla].networkStore).